tests/server/api/conftest.py

```diff
@@ -269,13 +269,63 @@ async def dataset_with_revisions(session):
     session.add(dataset_version_5)
     await session.flush()
 
+    dataset_version_6 = models.DatasetVersion(
+        id=6,
+        dataset_id=2,
+        description="datum gets created",
+        metadata_={},
+    )
+    session.add(dataset_version_6)
+    await session.flush()
+
+    dataset_version_7 = models.DatasetVersion(
+        id=7,
+        dataset_id=2,
+        description="datum gets deleted",
+        metadata_={},
+    )
+    session.add(dataset_version_7)
+    await session.flush()
+
+    dataset_version_8 = models.DatasetVersion(
+        id=8,
+        dataset_id=2,
+        description="datum gets created",
+        metadata_={},
+    )
+    session.add(dataset_version_8)
+    await session.flush()
+
+    dataset_version_9 = models.DatasetVersion(
```

Comment:
Suggestion: The bulk insert API can reduce the number of lines needed to define the fixture.

Reply:
good idea

Reply:
going to punt on this one until later